scripts/create-example-tests.js

```diff
@@ -1144,9 +1147,24 @@ function validateTest(testParsed, data, { addTestError = () => {} } = {}) {
  * @param {Queryable<AriaATParsed.Key>} data.key
  * @param {Queryable<{name: string, path: string}>} data.example
  * @param {Queryable<{at: string, mode: string, render: function({key: *}): string}>} data.modeInstructionTemplate
+ * @param {string} testPlanDirectory
  * @returns {AriaATFile.CollectedTest}
  */
-function collectTestData({ test, command, key, example, modeInstructionTemplate }) {
+function collectTestData(
+  { test, command, key, example, modeInstructionTemplate },
+  testPlanDirectory
+) {
+  const referencePageLocation = example.where({
+    name: test.setupScript ? test.setupScript.name : '',
+  });
+
+  if (!referencePageLocation) {
+    console.error(
+      `ERROR: Setup script ${test.setupScript.name} found in "${testPlanDirectory}/tests.csv" for "test id ${test.testId}: ${test.task}" not defined in "${testPlanDirectory}/reference".`
```

Comment:
There  is an existing error for this in `validateTest`.

```js
addTestError(`Setup script does not exist: ${testParsed.setupScript.name}`);
```

Reply:
Reused the existing function and updated the copy as `[Test 1]: Setup script does not exist: "setFocusOnButtonn" for "test id 1: trigger alert" not defined.`